scripts/release/generate_release_notes.sh

```diff
@@ -64,53 +64,78 @@ source "$SCRIPT_DIR/release/check_commit_metadata.sh" "${old_version}" "${ref}"
 # Sort commits by title prefix, then by date, only return sha at the end.
 mapfile -t commits < <(git log --no-merges --pretty=format:"%ct %h %s" "${old_version}..${ref}" | sort -k3,3 -k1,1n | cut -d' ' -f2)
 
-breaking_changelog=
-feat_changelog=
-fix_changelog=
-other_changelog=
+# From: https://github.com/commitizen/conventional-commit-types
+# NOTE(mafredri): These need to be supported in check_commit_metadata.sh as well.
+declare -a section_order=(
+	breaking
+	feat
+	fix
+	docs
+	refactor
+	perf
+	test
+	build
+	ci
+	chore
+	revert
+	other
+)
+
+declare -A section_titles=(
+	[breaking]='BREAKING CHANGES'
+	[feat]='Features'
+	[fix]='Bug Fixes'
+	[docs]='Documentation'
+	[refactor]='Code Refactoring'
+	[perf]='Performance Improvements'
+	[test]='Tests'
+	[build]='Builds'
+	[ci]='Continuous Integration'
+	[chore]='Chores'
+	[revert]='Reverts'
+	[other]='Other Changes'
+)
```

Comment:
I think most of this should be banished to a collapsible section at the bottom.

Reply:
Considering users will have to scroll down to the releases, maybe it wouldn't be such a bad idea. It's a bit hard to decide what's important and what's not, though. For instance, docs could be seen as either or. Likewise with perf, considering we rarely have those changes, it'd be a safe option to always show them.

I'd say show: breaking, feat, fix, docs, perf.

Others could potentially be hidden.

Then again, how long the notes are will largely depend on our release cadence. Usually we're fast(er) with new releases, so there's less need to hide.